git_machete/git_operations.py

```diff
@@ -865,7 +865,7 @@ def get_latest_checkout_timestamps(self) -> Dict[str, int]:  # TODO (#110): defa
                     result[to_branch] = int(match.group(1))
         return result
 
-    def get_commit_information(self, information: str, commit: Optional[AnyRevision] = None) -> str:
+    def get_commit_information(self, commit: AnyRevision, information: str) -> str:
         if information not in GIT_FORMAT_PATTERNS:
```

Comment:
Actually! while you're here, maybe instead of using raw strings like `"author date"` to distinguish b/w different pieces of information... let's use constants (enumeration?) instead, which will fail during mypy check, rather than in runtime